views/js/runner/config/qtiServiceConfig.js

```diff
@@ -46,7 +46,8 @@ define([
         testCompilation : true,
         serviceCallId : true,
         bootstrap : false,
-        timeout : false
+        timeout : false,
+        tenantName : false
```

Comment:
same as comment above, you should not pass plugin specific info or it will be mess soon